rest/servers/spring-boot/owasp-suppressions.xml

```diff
@@ -43,4 +43,12 @@
         <cve>CVE-2013-7315</cve>
         <cve>CVE-2014-0054</cve>
     </suppress>
+    <suppress>
+        <!-- Only present when JDOM 1.x or 2.x jar is in the classpath. JDOM has been excluded. -->
```

Comment:
❓ Same question as before. If we cannot be sure then maybe this should not be suppressed here.